### ISSUE TITLE
Accessibility: missing labels for inputs

### ISSUE THREAD

**Issue by Augs0** [AmirMehrabi/cssstyler#48]
All inputs should have labels or aria-labels to adhere to basic accessibility guidelines

**Comment by AmirMehrabi** [comment]
Thank you for the attention my man. Will take a look and have a fix ASAP.
